Sources/SwiftCrossUI/Views/Modifiers/TextSelectionModifier.swift

```diff
@@ -0,0 +1,12 @@
+extension View {
+    /// Set SwiftUI.Text selectability
```

Comment:
Change to:

```swift
/// Set selectability of contained text.
```

In future this modifier may affect other forms of text views such as markdown views, so it's best to keep the description abstract.